adapters/amqp-vertx/src/test/java/org/eclipse/hono/adapter/amqp/VertxBasedAmqpProtocolAdapterTest.java

```diff
@@ -264,6 +268,13 @@ private MessageSender givenATelemetrySenderForAnyTenant() {
         return sender;
     }
 
+    private Device getAuthenticatedDevice() {
+        final Device authenticatedDevice = mock(Device.class);
```

Comment:
Is there a reason why you want to create a mock instead of just instantiating a `new Device(TEST_TENANT_ID, TEST_DEVICE)`?

Reply:
:+1: